### PR TITLE
SEEDs Page

### LINKED ISSUE #925: Seeds Page
### What would you like to be added?

We need a page with all the info regarding Seeds:

- [What they are](https://wiki.metagame.wtf/docs/how-does-it-work/xp/)
- [Where you can claim them](https://claim.metagame.wtf/)
- [Where you can buy them](https://balancer.exchange/#/swap/ether/0x30cf203b48edaa42c3b4918e955fed26cd012a3f)
  - Along with the [Patron's path](https://meta-game.notion.site/Patron-Path-1db90c8bb4c84398a6fe1f672ea5e855)
- [Where you can plant them](https://pools.balancer.exchange/#/pool/0xea05a15dbce2eb543ffda16950e95b2bd2e40d0e/)
- [What you can do with them](https://meta-game.notion.site/Usefulness-of-Seeds-56358863654746738052ab2e1a8fdade)
  - Some of these usecases are live, we should make them easy to use

To begin with, it can be a simple page with short explanations & links to where you can read more or do actions such as buy, claim or plant.
The design of the page is the first thing needed. Need help finding a designer, a writer or writing & designing it yourself? [Ask about it here](https://discord.gg/4gfpHGgUvE)

Longer term, actions such as claim, buy/sell & plant should all be directly integrated into the page in a similar way that the [PrimeDAO did it](https://github.com/MetaFam/TheGame/issues/263) for their own balancer pool:
image

### Why is this needed?

Because right now info about Seeds & all of the above actions are dispersed all over the place and the "Seeds" link in the megamenu simply takes people to the "XP & Seeds" page on github.

### LINKED ISSUE #1074: [MetaMenu] - Disclaimer in the Invest section
### What would you like to be added?

The "Invest" section of the metamenu should feature a disclaimer saying something like "We offer no prospects of financial return, invest for good feels only."

### Why is this needed?

To avoid any legal ramifications of MetaGame offering investments or Seeds being labeled a security.

### PR DESCRIPTION
**Pull request by vidvidvid**
## Overview

**What features/fixes does this PR include?**

Page with seed info.
1. Removed all `Invest` sections, merged them into one - `Seeds` and placed it along other topics in the `Contribute` section.
2. Copied the content and design from the [notion page](https://www.notion.so/meta-game/Seeds-3a301b30f9394751b77b2450f7ae2329)
3. Set max width of the site as per the [Figma design](https://www.figma.com/file/h4w7a6N8qtDblQDO4UpfkQ/Seeds-page?node-id=718%3A22897)

**Please provide the GitHub issue number**

Closes #925 
Closes #1074 

#### Mobile wip:

seeds_footer

### CODE REVIEW

File: packages/web/components/Seeds/modals.tsx [hunk 1]
```diff
@@ -0,0 +1,750 @@
...
+
+      <Box onClick={() => setIsOpenCap(!isOpenCap)} my={4}>
+        <details>
+          <summary>What’s market cap?</summary>
+        </details>
+      </Box>
+      {isOpenCap && (
+        <Text>
+          At the time of writing this post (February 2022), the market cap was
+          $600k
```
Comment (alalonde) [inline packages/web/components/Seeds/modals.tsx:745]: Can we just link out to this calculation? Our market cap on mainnet is higher than this already

File: packages/web/pages/seeds.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            >
+              Seeds
+              <Text pl={4} alignSelf="center" fontSize="4xl">
+                🌱
+              </Text>
+            </Heading>
+            <Text mb={4}>Seeds are MetaGame's labor token.</Text>
+            <Text mb={4}>
+              People contribute towards creation of MetaGame, meanwhile
+              generating XP & getting paid out on their XP generated - in Seeds.
```
Comment (alalonde) [inline packages/web/pages/seeds.tsx:64]:
```suggestion
              generating XP & getting paid out in Seeds proportional to their gained XP
```

File: packages/web/utils/menuLinks.ts [hunk 3]
```diff
@@ -26,10 +26,7 @@ export const descriptions = {
   thegreathouses:
     'The Great Houses are here to give people a meta view of different areas of interest',
   wiki: 'New to MetaGame? This is the first thing you should dive deep into',
-  becomeapatron: "Don't have time to play? Support MetaGame by buying Seeds",
-  buyseeds:
-    'Find out all about the SEEDs; why, where & how you should buy them',
-  plantseeds: 'Plant your SEEDs and watch them grow',
+  seeds: 'Find out all about the SEEDs; why, where & how you should buy them',
```
Comment (alalonde) [inline packages/web/utils/menuLinks.ts:29]:
```suggestion
  seeds: 'Find out about SEEDs -- why, where & how you should buy them',
```

**Review by alalonde — CHANGES_REQUESTED** [review]
Nice work! I added a whole bunch of copy edits.

File: packages/web/components/Seeds/modals.tsx (hunk 1 above)
Comment (peth-yursick) [reply packages/web/components/Seeds/modals.tsx:745]: The problem is that even the "real" calculation is wrong 😂
Displaying $861k at the price of $15.59 but the real price is more like $11..
image

File: packages/web/components/Seeds/modals.tsx [hunk 4]
```diff
@@ -0,0 +1,750 @@
...
+        mr={1}
+      >
+        GitHub
+      </Link>
+      - your XP automagically accrues. Once a month, you are rewarded Seeds for
+      your contributions, based on the amount of XP that you generated.
+    </Text>
+
+    <Text mb={2}>
+      Your XP is calculated by a thing called
```
[the remark this answers is not in the document]
Comment (peth-yursick) [reply packages/web/components/Seeds/modals.tsx:47]: This one was intentional actually, metagame's quirky voice

**Comment by peth-yursick** [comment]
Thank you for all the corrections @alalonde 🙏

File: packages/web/components/Seeds/modals.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (vidvidvid) [reply packages/web/components/Seeds/modals.tsx:47]: sooo do we leave it or not?

File: packages/web/components/Seeds/modals.tsx [hunk 5]
```diff
@@ -0,0 +1,750 @@
...
+        <ul>
+          <ListItem ml={4}>
+            The minimal requirement to join as a patron is 8pSeeds which is
+            roughly $150.
+          </ListItem>
+        </ul>
+        <ul>
+          <ListItem ml={4}>
+            Beyond that, you are free to water them however much you want if you
+            wish a high position in the ranked league of patrons, but be careful
```
[the remark this answers is not in the document]
Comment (vidvidvid) [reply packages/web/components/Seeds/modals.tsx:551]: do you have the link? not sure where to find it.

**Comment by vidvidvid** [comment]
Thank you for the thorough review, @alalonde ! I fixed all of the straightforward things, but there are still a couple of pending points. I also need to finish up the design which dave posted today: https://www.figma.com/file/h4w7a6N8qtDblQDO4UpfkQ/Seeds-page?node-id=0%3A1

File: packages/web/components/Seeds/modals.tsx (hunk 1 above)
Comment (vidvidvid) [reply packages/web/components/Seeds/modals.tsx:769]: can you gimme the link?

File: packages/web/components/Seeds/modals.tsx [hunk 6]
```diff
@@ -0,0 +1,750 @@
...
+          <summary>💸 Spending Seeds</summary>
+        </details>
+      </Box>
+      {isOpenSpending && (
+        <Flex direction="column">
+          <Text fontSize={16} fontWeight="bold" my={2}>
+            Things you can spend on now:
+          </Text>
+          <Box p={2} bgColor="whiteAlpha.300" mb={2}>
+            💡 Note: You need to be a member to access any of these
```
[the remark this answers is not in the document]
Comment (vidvidvid) [reply packages/web/components/Seeds/modals.tsx:163]: @peth-yursick

File: packages/web/components/Seeds/modals.tsx (hunk 1 above)
Comment (alalonde) [reply packages/web/components/Seeds/modals.tsx:770]: https://etherscan.io/token/0x30cf203b48edaa42c3b4918e955fed26cd012a3f 

This is on mainnet however so it's not really relevant. Might as well just leave it as-is

Comment (peth-yursick) [reply packages/web/components/Seeds/modals.tsx:770]: https://etherscan.io/token/0x30cf203b48edaa42c3b4918e955fed26cd012a3f
https://www.coingecko.com/en/coins/metagame

File: packages/web/components/Seeds/modals.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (peth-yursick) [reply packages/web/components/Seeds/modals.tsx:565]: https://my.metagame.wtf/patrons

File: packages/web/components/Seeds/modals.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (peth-yursick) [reply packages/web/components/Seeds/modals.tsx:163]: right.. maybe better changed to "You need to be a Player or Patron to access any of these"

File: packages/web/components/Seeds/modals.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (alalonde) [reply packages/web/components/Seeds/modals.tsx:565]: /community/patrons

**Review by alalonde — APPROVED** [review]

File: packages/web/components/Seeds/modals.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (peth-yursick) [reply packages/web/components/Seeds/modals.tsx]: I'd say yes but I don't have a strong preference
probably about time we made a "metagame voice/tone" guide for people writing things & to make these decisions easier..

File: packages/web/components/Seeds/modals.tsx [hunk 7]
```diff
@@ -0,0 +1,750 @@
...
+      )}
+
+      <Box onClick={() => setIsOpenCap(!isOpenCap)} my={4}>
+        <details>
+          <summary>What’s market cap?</summary>
+        </details>
+      </Box>
+      {isOpenCap && (
+        <Text>
+          At the time of writing this post (February 2022), the market cap was
```
[the remark this answers is not in the document]
Comment (peth-yursick) [reply packages/web/components/Seeds/modals.tsx:769]: yeah, maybe should be changed to "MetaGame's market cap"..
also the "at the time of writing" part should be removed if we're going the calculation way

**Pull request merged**